lib/ui/common/app_colors.dart

```diff
@@ -8,7 +8,11 @@ const Color kcLightGrey = Color.fromARGB(255, 187, 187, 187);
 const Color kcVeryLightGrey = Color(0xFFE3E3E3);
 const Color kcBackground = Color(0xFFFFFFFF);
 const Color kcIconBackground = Color(0x4BFFF3CE);
+const Color kcIconBackground2 = Color(0xFFF5F5F5);
+const Color kcIconBackground3 = Color(0x728B8B8B);
 const Color kcBackgroundColor = kcDarkGreyColor;
+const Color kBarrierColor = Color(0x99454545);
+const Color kcOrangeShade = Color(0xFFFFF3EC);
```

Comment:
kindly move these colors that you created inside the app colors class

Reply:
Ohh, I met some in the app like that, so I thought it was allowed.
I'd remove it